### PR TITLE
Update ansible_playbook_to_role util for RHEL 9

### PR DESCRIPTION
**Pull request by dmc5179**
#### Description:
Update ansible_playbook_to_role util for RHEL 9
Switch default branch from master to main
Remove unsupported characters in role names during conversion process; i.e [ and ]

#### Rationale:

Util needs to be updated to support converting the ansible playbooks from this repo into ansible roles as has been done for RHEL 7 and RHEL 8.

#### Review Hints:

After building this main content repo, the following command is used to convert the playbooks into roles, which needs the added RHEL 9 support

```
./utils/ansible_playbook_to_role.py --product rhel9 --profile $p --organization RedHatOfficial --tag-release --token ${GIT_TOKEN} --build-playbooks-dir ./build/ansible/
```

I have used the updated version of the script for the last few content releases but did not have a chance until now to submit a PR.

### CODE REVIEW

**Comment by Mab879** [comment]
Please resolve the merge conflicts.

File: utils/ansible_playbook_to_role.py [hunk 1]
```diff
@@ -399,8 +400,10 @@ def save_to_disk(self, directory):
         for filename in self.PRODUCED_FILES:
             abs_path = os.path.join(directory, self.name, filename)
             mkdir_p(os.path.dirname(abs_path))
+            print("role dir: %s", os.path.dirname(abs_path))
             open(abs_path, 'wb').write(self.file(filename).encode("utf-8"))
-
+            # Ansible Galaxy won't accept [ or ] characters
+            sh.sed('-i','s/\[DRAFT\]/DRAFT/g',abs_path)
```
Comment (Mab879) [inline utils/ansible_playbook_to_role.py:406]:
```suggestion
            sh.sed('-i','s/\[DRAFT\]/DRAFT/g', abs_path)
```

File: utils/ansible_playbook_to_role.py [hunk 2]
```diff
@@ -399,8 +400,10 @@ def save_to_disk(self, directory):
...
 
 class RoleGithubUpdater(object):
```
Comment (Mab879) [inline utils/ansible_playbook_to_role.py:408]:
```suggestion

class RoleGithubUpdater(object):
```

**Review by Mab879 — CHANGES_REQUESTED** [review]
Please fix the the PEP8 issues and remove the merge commit from this branch.

**Comment by dmc5179** [comment]
@Mab879 I'm not sure how to do that last part, remove the merge commit from this branch.

**Comment by Mab879** [comment]
> @Mab879 I'm not sure how to do that last part, remove the merge commit from this branch.

First all, thanks for the work.

Reviewing the changes I wondering if we can just close this PR.

The [DRAFT] to DRAFT change should no longer be needed as we now use "DRAFT - " as the prefix.

So that only leaves that print statement.

If you want that print statement, with all the recent changes it might be best just to open a new PR based on the latest master.

Again, thanks for the your work and happy to help.

**Comment by Mab879** [comment]
Thanks again for this PR.

However, I don't think the changes are necessary.

Please feel free to open a new PR to make additional changes.

**Pull request closed without merging**